### ISSUE TITLE
.send_keys predominately fails with IEDriverServer v3.3.0 (32bit), but ok with v3.3.0 (64bit)

### ISSUE THREAD

**Issue by russcarter** [SeleniumHQ/selenium#3761]
## Meta -
OS:  Win7 Pro SP1 64bit
Selenium Version:  3.3.2
Bindings: Python v2.7.12 32bit or v3.5.2 32bit
Browser:  IE 
Browser Version:  11.0.9600.18617 (KB4012204)

## Expected Behavior -
.send_keys consistently works to populate file path and name into file uplodad dialog box (using latest IEDriverServer)
## Actual Behavior -
.send_keys predominately fails to populate file path and name into file upload dialog box using IEDriverServer v3.3.0 (32bit) and v3.3.99.4.  (By predominately fails, I'm saying it fails 19 out of 20 times, maybe 1 in 20 it works ok)   In contrast, .send_keys seems consistently good using IEDriverServer v2.48 and v3.3.0 (64bit).

In a nutshell, the IE file upload dialog box hangs, with no path / file name entry, coincident with the .send_keys call.

## Steps to reproduce -

`from selenium import webdriver
driver = webdriver.Ie()
# lets demonstrate this on a FileUpload page
driver.get('https://www.websupergoo.com/file-upload-1.htm')
# find the 'Browse' button
page_upload_button = driver.find_element_by_name("fileField")
# define a path and file name to upload local Win7 OS.
file_to_upload = "C:\\foo.bar"
# try to FileUpload to illustrate the cited problem behavior. FileUpload just hangs here
page_upload_button.send_keys(file_to_upload)`

**Comment by Rhinomcd** [comment]
This was talked about at the final panel of SeleniumConf 2017(Austin).  Jim Evans referenced his blog post here going into detail why this happens.
http://jimevansmusic.blogspot.com/2014/09/screenshots-sendkeys-and-sixty-four.html

Basically there is an issue with windows hooks and the way IE 10 and 11 Operates.

Here's a link to the timestamp where he answers this https://youtu.be/OiCBCnmr4cA?t=38m

**Comment by jimevans** [comment]
This issue as described is a duplicate of #3587. If you believe this to be a distinctly different issue, feel free to request it to be reopened, or to open another new issue.
